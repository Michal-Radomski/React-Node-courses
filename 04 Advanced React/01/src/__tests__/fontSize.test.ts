import fontSize from "../fontSize";

test("snapshot of fontsizes", (): void => {
  expect(fontSize).toMatchSnapshot();
});
