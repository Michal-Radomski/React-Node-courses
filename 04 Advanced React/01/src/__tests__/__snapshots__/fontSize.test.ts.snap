// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`snapshot of fontsizes 1`] = `
{
  "base": "base",
  "lg": "lg",
  "sm": "sm",
  "xl": "xl",
  "xs": "xs",
}
`;
